code/game/gamemodes/cult/runes.dm

```diff
@@ -818,7 +838,7 @@ var/list/teleport_runes = list()
 						 "<span class='cultitalic'>You channel your life energy into [src], [density ? "preventing" : "allowing"] passage above it.</span>")
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
-		C.apply_damage(2, BRUTE, pick("l_arm", "r_arm"))
+		C.apply_damage(4, BRUTE, pick("l_arm", "r_arm"))
 
 
 //Rite of Joined Souls: Summons a single cultist.
```

Comment:
Maybe stop this working if the cultist is cuffed? 

We've had issues with teleporting cultists that we have to lethal to subdue. It sucks when a cultist is cuffed, chaired in the brig - and they just get teleported out.